### DISCUSSION TITLE
The method of masking edge

### DISCUSSION THREAD

**Discussion by StefanIsSmart** [pyg-team/pytorch_geometric#9094] [Q&A]
For example,  Some graph is directed (like [1,0],[0,1]), and I want to mask the double directed edges together randomly.
How could I do that? Is there any function?

**Comment by rusty1s** [comment]
```python
mask = (edge_index[0] == node_id) & (edge_index[1] == node_id)
edge_index = edge_index[:, ~mask]
```
